erpnext/accounts/doctype/account/account.py

```
@@ -23,13 +23,73 @@ class InvalidAccountMergeError(frappe.ValidationError):
+	# This code is auto-generated. Do not modify anything in this block.
+
+	from typing import TYPE_CHECKING
+
+	if TYPE_CHECKING:
+		from frappe.types import DF
+
+		account_currency: DF.Link | None
+		account_name: DF.Data
+		account_number: DF.Data | None
+		account_type: DF.Literal[
+			"",
```

Ruff (0.14.8)

Syntax error in forward annotation: Expected an expression

(F722)